packages/iov-keycontrol/src/wallets/ed25519wallet.spec.ts

```diff
@@ -92,9 +92,23 @@ describe("Ed25519Wallet", () => {
     expect(newIdentity4.label).toEqual(lastIdentity.label);
   });
 
-  it("throws when adding the same keypair twice", async () => {
-    // Same keypair leads to the same identity identifier, so we don't support it
+  it("can create different identities with the same keypair", async () => {
```

Comment:
Good test, thought of the same thing when making that comment.
That every identity is tied to a chain, and we *encourage* using different keys for each chain, but nothing *prohibits* the reuse